CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Reduce memory usage when committing write transactions.
+* Improve performance of compacting encrypted Realm files.
+* Throw an exception when opening a Realm file with an invalid top ref rather
```

Comment:
don't think users know what a "top ref" is.

Reply:
This is an `InvalidDatabase` C++ exception now. How is that propagated to end users? Could they catch it and do something meaningful like deleting the database and continue? This has been heavily requested in some issue i fI recall correctly?

Reply:
It surfaces as an untyped exception without a unique error code. Without further SDK-level work there isn't a good way to handle just this exception.